File: 백준/Bronze/13866. 팀 나누기/팀 나누기.py
import sys
input = sys.stdin.readline
from collections import deque

a = list(map(int, input().split()))
a.sort()
a = deque(a)

d = []
while a:
    b = a.popleft()
    c = a.pop()
    d.append(b+c)

print(abs(d[0]-d[1]))

File: 백준/Bronze/13866. 팀 나누기/팀 나누기.js
const input = require("fs")
  .readFileSync(process.platform === "linux" ? "/dev/stdin" : "./example.txt")
  .toString()
  .trim()
  .split(" ")
  .map(Number);

input.sort((a, b) => a - b);

const res1 = input.splice(1, 2).reduce((acc, cur) => acc + cur, 0);
const res2 = input.reduce((acc, cur) => acc + cur, 0);

console.log(Math.abs(res1 - res2));
